pkg/gardenlet/controller/controllerinstallation/controllerinstallation/reconciler.go

```diff
@@ -357,6 +357,15 @@ func (r *Reconciler) delete(
 
 	conditionInstalled = v1beta1helper.UpdatedConditionWithClock(r.Clock, conditionInstalled, gardencorev1beta1.ConditionFalse, "DeletionSuccessful", "Deletion of old resources succeeded.")
 
+	gardenClusterServiceAccount := &corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{
+		Name:      v1beta1constants.ExtensionGardenServiceAccountPrefix + controllerInstallation.Name,
+		Namespace: gardenerutils.ComputeGardenNamespace(r.Config.SeedConfig.Name),
```

Comment:
We already have the `Seed` object, retrieved based on `controllerInstallation.Spec.SeedRef.Name`.

Should we use it for consistency with the `reconcile` function?
```suggestion
		Namespace: gardenerutils.ComputeGardenNamespace(seed.Name),
```